_includes/headtags.html

```diff
@@ -3,17 +3,17 @@
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
 {% if page.excerpt != blank %}
-    {% assign description = page.excerpt %}
+{% assign description = page.excerpt %}
 {% elsif page.description != blank %}
-    {% assign description = page.description %}
+{% assign description = page.description %}
 {% else %}
-    {% assign description = site.description %}
+{% assign description = site.description %}
 {% endif %}
 
 {% if page.layout == "archive" %}
-    {% if site.data.descriptions[page.autopages.display_name] != blank %}
-        {% assign description = site.data.descriptions[page.autopages.display_name]  %}
-    {% endif %}
+{% if site.data.descriptions[page.autopages.display_name] != blank %}
+{% assign description = site.data.descriptions[page.autopages.display_name] %}
+{% endif %}
 {% endif %}
```

Comment:
VS Code doesn't preserve the liquid formatting inside HTML files, can you replace this text with the original indentation for better readability?

Reply:
done